src/base/gps.cc

```diff
@@ -120,4 +120,79 @@ std::vector<Eigen::Vector3d> GPSTransform::XYZToEll(
   return ell;
 }
 
+std::vector<Eigen::Vector3d> GPSTransform::EllToENU(
+    const std::vector<Eigen::Vector3d>& ell) const {
+  // Convert GPS (lat / lon / alt) to ECEF
+  std::vector<Eigen::Vector3d> xyz = EllToXYZ(ell);
+
+  // GPS reference origin
+  const double lat0 = ell[0](0);
+  const double lon0 = ell[0](1);
+
+  return XYZToENU(xyz, lat0, lon0);
+}
+
+std::vector<Eigen::Vector3d> GPSTransform::XYZToENU(
```

Comment:
I don't understand how this is meant to be used. The first point is used as the reference but then we require the input of the same point also as lat0/lon0?

Reply:
I guess this is solved by the change in EllToENU function now explicitly requiring the origin lat/lon coords?

Reply:
LGTM, thanks.